docs/troubleshooting.md

```diff
@@ -114,6 +114,42 @@ Available in Rancher v2.6.3 (Fleet v0.3.8), the ability to enable debug logging
 - Select **Apps & Marketplace**, then **Installed Apps** from the dropdown
 - From there, you will upgrade the Fleet chart with the value `debug=true`. You can also set `debugLevel=5` if desired.
 
+#### Via Fleet Install Options
```

Comment:
Could this be linked or somehow unified with information available on the `ref-configuration.md` page?